packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -41,41 +38,42 @@ export default function ListViewSidebar() {
 		}
 	}
 
-	const [ tab, setTab ] = useState( 'list-view' );
-
 	// This ref refers to the sidebar as a whole.
 	const sidebarRef = useRef();
-	// This ref refers to the list view tab button.
-	const listViewTabRef = useRef();
-	// This ref refers to the outline tab button.
-	const outlineTabRef = useRef();
+	// This ref refers to the tab panel.
+	const tabPanelRef = useRef();
 	// This ref refers to the list view application area.
 	const listViewRef = useRef();
 
+	// Must merge the refs together so focus can be handled properly in the next function.
+	const listViewContainerRef = useMergeRefs( [
+		contentFocusReturnRef,
+		focusOnMountRef,
+		listViewRef,
+	] );
+
 	/*
 	 * Callback function to handle list view or outline focus.
 	 *
-	 * @param {string} currentTab The current tab. Either list view or outline.
-	 *
 	 * @return void
 	 */
-	function handleSidebarFocus( currentTab ) {
-		// List view tab is selected.
-		if ( currentTab === 'list-view' ) {
-			// Either focus the list view or the list view tab button. Must have a fallback because the list view does not render when there are no blocks.
-			const listViewApplicationFocus = focus.tabbable.find(
-				listViewRef.current
-			)[ 0 ];
-			const listViewFocusArea = sidebarRef.current.contains(
-				listViewApplicationFocus
-			)
-				? listViewApplicationFocus
-				: listViewTabRef.current;
-			listViewFocusArea.focus();
-			// Outline tab is selected.
-		} else {
-			outlineTabRef.current.focus();
+	function handleSidebarFocus() {
```

Comment:
FYI, if we wanted to still use the "selected tab" as an argument to this function, we could use the `onSelect` prop on `TabPanel` to get updates on which is the currently selected tab.
That would probably make the code in the `handleSidebarFocus` less hacky and easier to read